bigquery_etl/query_scheduling/formatters.py

```diff
@@ -19,10 +19,16 @@ def format_schedule_interval(interval):
 def format_attr(d, attribute, formatter_name):
     """Apply a formatter to a dict attribute."""
     for name in dir(query_scheduling.formatters):
+        if name != formatter_name:
```

Comment:
Had to make this tweak here because for some reason the same date attribute was being put through the formatted function twice which resulted in a TypeError inside `format_date` function because the second around the start_date value was already a datetime object. Alternatively, we could adjust the `format_date` function to accept both string and datetime and just return the value if already of datetime type.